x-pack/plugins/alerting/server/task_runner/create_execution_handler.ts

```diff
@@ -114,13 +68,20 @@ export function createExecutionHandler<
     actionSubgroup,
     context,
     state,
+    alertExecutionStore,
     alertId,
   }: ExecutionHandlerOptions<ActionGroupIds | RecoveryActionGroupId>) => {
-    const triggeredActions: AlertAction[] = [];
     if (!ruleTypeActionGroups.has(actionGroup)) {
       logger.error(`Invalid action group "${actionGroup}" for rule "${ruleType.id}".`);
-      return triggeredActions;
+      alertExecutionStore.completion = ActionsCompletion.PARTIAL;
```

Comment:
Here is another question:
We stop the alert execution if there is no actionGroup.
Should we consider this as PARTIAL completion or COMPLETE?

Reply:
This is typically an error on the rule type creator end where they schedule an action for an alert with an invalid action group id. We want to avoid executing this action but I don't think we want to bubble it up to the user with a warning because there's nothing they can do about it.